src/website/shared/listeners/cache_suggestions.py

```diff
@@ -295,3 +301,32 @@ def parse_drv_name(name: str) -> tuple[str, str]:
         return match.group(1), match.group(2)
     else:
         return name, ""
+
+
+def maintainers_list(packages: dict, edits: list[MaintainersEdit]) -> list[dict]:
+    """
+    Returns a deduplicated list (by GitHub ID) of all the maintainers of all the
+    affected packages linked to this suggestion, modified by potential
+    user-supplied edits.
+    """
+
+    to_remove = {
+        m.maintainer for m in edits if m.type == MaintainersEdit.EditType.REMOVE
+    }
```

Comment:
```suggestion
    to_remove = [m.maintainer for m in edits if m.type == MaintainersEdit.EditType.REMOVE]
```

Was this the linter trying to be clever?